tests/components/mqtt/test_init.py

```diff
@@ -91,16 +87,19 @@ def test_publish_calls_service(self):
         self.hass.block_till_done()
 
         assert len(self.calls) == 1
-        assert self.calls[0][0].data["service_data"][mqtt.ATTR_TOPIC] == "test-topic"
-        assert (
-            self.calls[0][0].data["service_data"][mqtt.ATTR_PAYLOAD] == "test-payload"
-        )
+        assert self.calls[0][0].data["service_data"][
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)